float64[6] pwm
---
float64[6] out
